wiki/Rules/Visual_Content_Considerations/en.md

```diff
@@ -1,117 +1,52 @@
 
-This also includes "flirty" depictions which are - in good faith - considered as playful, not unreasonably excessive, and not in violation of any of the other inappropriate content markers.
+- **sexual content involving or targeted at minors**
+- **excessive violence, gore, dismemberment, decapitation, or maiming**
+- **significant sexual innuendo**
+- **sexual posturing or solicitation**
+- **erotic content or graphic displays of sexuality**
+- **examples of drug use, preparation, or identification with illicit drug subculture**
+- **depictions of deliberately inflammatory political, cultural, religious, or social content**
 
-## Further considerations
+## Exceptions
 
-The anecdotal "rule of thumb" is that one should generally be comfortable with their family members or other judgemental authority figure viewing the content and be capable of adequately explaining what is going on in it.
+Images or visual elements containing examples of the following may be considered acceptable:
 
-This is by no means a strict requirement, but it is a good way to help inform your decisions on what is and isn't appropriate.
+- mild to moderate depictions of violence (small amounts of blood, non-critical or superficial wounding)
+- artistic nudity (in the context of classical artwork, or work related to the element, such as album art with minor nudity present)
+- mild romance, flirting or intimate affection (hugging, embracing, kissing without sexual connotations, hand-holding, etc.)
+- swimsuits of reasonable design (standard swimwear with general coverage, ordinary bikini)
 
-### What do I do if I am uncertain about a piece of visual content?
+## Getting your image assessed
 
-In situations whether the appropriateness of a given piece of visual content is unclear, the mapper must submit a formal request to any Beatmap Nominator for further appraisal, who will then refer the issue to the Global Moderation Team to decide based on a supermajority (>70%) consensus.
+If you are unsure about an image that is going to be included in a beatmap in any way, **please submit a formal request to any [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators) or member of the [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team).**
 
-Once a consensus is met and issued, the resulting ruling on its appropriateness is final and can not be challenged under any circumstances, unless edits are made to try and fix the contentious issues cited by the ruling. Any number of these subsequent requests may be made within reason, so long as a good faith effort is being made to try and fix the issues listed.
+The image will be raised to a vote that must pass by a considerable majority, or will be automatically denied. The result of any vote is **final** and may not be overridden or raised for discussion again, unless the image is significantly altered in an attempt to resolve the problems with it.
```

Comment:
while I understand the first sentence, it reads like it's missing "it" in the last part

Reply:
I was thinking about this too, unfortunately it's still ambiguous whether "it" would mean the image or the image's vote. and if you interpret that as "the image", the sentence reads like: "it will be raised to a vote [...] OR will be automatically denied"

I think this could be better as two sentences, one to say there's a vote, another to say what happens when it passes/fails

Reply:
altered the wording here